src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -1,48 +1,45 @@
 import { BDD_FORM_ROOT_URL } from 'applications/disability-benefits/bdd/constants';
-import hlrManifest from 'applications/disability-benefits/996/manifest.json';
-
-import hcaConfig from 'applications/hca/config/form.js';
-import dependentStatusConfig from 'applications/disability-benefits/686/config/form';
-import feedbackConfig from 'applications/edu-benefits/feedback-tool/config/form.js';
-import burialsConfig from 'applications/burials/config/form.js';
+import edu0993Config from 'applications/edu-benefits/0993/config/form.js';
+import edu0993Manifest from 'applications/edu-benefits/0993/manifest.json';
+import edu0994Config from 'applications/edu-benefits/0994/config/form.js';
+import edu0994Manifest from 'applications/edu-benefits/0994/manifest.json';
 import edu1990Config from 'applications/edu-benefits/1990/config/form.js';
-import edu1995Config from 'applications/edu-benefits/1995/config/form.js';
+import edu1990Manifest from 'applications/edu-benefits/1990/manifest.json';
 import edu1990eConfig from 'applications/edu-benefits/1990e/config/form.js';
+import edu1990eManifest from 'applications/edu-benefits/1990e/manifest.json';
 import edu1990nConfig from 'applications/edu-benefits/1990n/config/form.js';
+import edu1990nManifest from 'applications/edu-benefits/1990n/manifest.json';
+import edu1995Config from 'applications/edu-benefits/1995/config/form.js';
+import edu1995Manifest from 'applications/edu-benefits/1995/manifest.json';
 import edu5490Config from 'applications/edu-benefits/5490/config/form.js';
+import edu5490Manifest from 'applications/edu-benefits/5490/manifest.json';
 import edu5495Config from 'applications/edu-benefits/5495/config/form.js';
-import edu0993Config from 'applications/edu-benefits/0993/config/form.js';
-import edu0994Config from 'applications/edu-benefits/0994/config/form.js';
-import preneedConfig from 'applications/pre-need/config/form.jsx';
+import edu5495Manifest from 'applications/edu-benefits/5495/manifest.json';
+import feedbackConfig from 'applications/edu-benefits/feedback-tool/config/form.js';
+import feedbackManifest from 'applications/edu-benefits/feedback-tool/manifest.json';
+import hcaConfig from 'applications/hca/config/form.js';
+import hcaManifest from 'applications/hca/manifest.json';
 import pensionConfig from 'applications/pensions/config/form.js';
-import disability526Config from 'applications/disability-benefits/526EZ/config/form.js';
-import bddConfig from 'applications/disability-benefits/bdd/config/form.js';
-import hlrConfig from 'applications/disability-benefits/996/config/form';
+import pensionManifest from 'applications/pensions/manifest.json';
+import preneedConfig from 'applications/pre-need/config/form.jsx';
+import preneedManifest from 'applications/pre-need/manifest.json';
+import { isPlainObject } from 'lodash';
+import { VA_FORM_IDS } from 'platform/forms/constants.js';
+import recordEvent from 'platform/monitoring/record-event';
+import React from 'react';
```

Comment:
Would you please move these imports back into their original order... I still think React should be the first thing we import

Reply:
> Would you please move these imports back into their original order... I still think React should be the first thing we import

It keeps reorganizing them when I save them in a different order :sweat_smile: Not sure how to disable this. I tried using `git add -p` but it combined my additions with the import reordering.  Is there a way to go line by line and accept/reject ?

Reply:
- you can use [git interactive](https://git-scm.com/book/en/v2/Git-Tools-Interactive-Staging) e.g. `git add -p`. Specifically you can use `n` to skip staging a chunk or `s` to split a chunk before choosing `y` or `n`

```
Stage this hunk [y,n,a,d,/,j,J,g,e,?]? ?
y - stage this hunk
n - do not stage this hunk
a - stage this and all the remaining hunks in the file
d - do not stage this hunk nor any of the remaining hunks in the file
g - select a hunk to go to
/ - search for a hunk matching the given regex
j - leave this hunk undecided, see next undecided hunk
J - leave this hunk undecided, see next hunk
k - leave this hunk undecided, see previous undecided hunk
K - leave this hunk undecided, see previous hunk
s - split the current hunk into smaller hunks
e - manually edit the current hunk
? - print help
```
- assuming you're using vscode you can turn off all of your plugins but `ESlint`

Reply:
> It keeps reorganizing them when I save them in a different order 😅 Not sure how to disable this. I tried using git add -p but it combined my additions with the import reordering. Is there a way to go line by line and accept/reject ?

Is that due to a VS Code extension or something? First I've heard of this

Reply:
Cool, it looks like that solved it!

Reply:
there is a on save hook that you can set a boolean for to stop this. We should set that to false. 😄